aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js

```diff
@@ -23,21 +23,19 @@ export const NameValueListEditExample = () => {
 
   return (
     <Box gap="medium">
-      <Box justify="between" width="large" direction="row">
-        <Heading level={2} size="small" margin="none">
-          Profile Details
-        </Heading>
-        {!edit && (
-          <Box justify="center">
+      <PageHeader
+        title="Profile Details"
+        actions={
+          !edit && (
             <Button
               alignSelf="center"
               secondary
               label="Edit Profile"
```

Comment:
this button gets removed on mobile I don't think that should be the case

Reply:
<img width="518" alt="Screen Shot 2022-05-04 at 5 36 36 PM" src="https://user-images.githubusercontent.com/42451602/166842077-0f3c4ba2-b1c8-44de-993d-3ff49086e8b1.png">
also now on smaller sizes it gets cut off

Reply:
Done.